Userland/Libraries/LibGfx/Rect.h

```diff
@@ -557,6 +557,24 @@ class Rect {
         return points;
     }
 
+    template<typename U = T>
+    [[nodiscard]] Gfx::Rect<U> scaled_to(Gfx::Rect<T> const& to, float pct) const
```

Comment:
`pct` (percentage) is not really obvious to me; not sure what would be better here.

weʼre assuming here linear transformation with ‘timeʼ varying from 0 to 1; maybe we could make that explicit? (i.e. by ‘scaled linearly toʼ?) and use `progress`, `time` or `at` as a parameter name? 🤔

Reply:
hmm… yeaaah… 🤔 sounds about right! 😁

(maybe even `factor` could be a thing—yet go with your proposal!)

Reply:
(btw. what do you think about explicit naming?)